src/c_api/c_api.cc

```diff
@@ -136,6 +136,8 @@ int MXSetNumOMPThreads(int thread_num) {
   API_END();
 }
 
+int MXGetVersion() { return static_cast<int>(MXNET_VERSION); }
```

Comment:
the return value is for error status.
Use int *out instead of return value.

Reply:
Ok, fixed.